src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,265 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService service;
+    private static StorageDao storageDao;
+    private static final String EMPTY_LINE = "";
+    private static final String ONE_SYMBOL_LINE = "a";
+    private static final String THREE_SYMBOLS_LINE = "abc";
+    private static final String FIVE_SYMBOLS_LINE = "12345";
```

Comment:
Ok this is 5 symbols line and what? what context this name gives to a code? Maybe it is better to name INVALID_SOMETING_FIVE_CHARS, to clearly show that this is something invalid